boot/debian_12/README.md

```diff
@@ -0,0 +1,23 @@
+# Debian 12
+
+- [Repo](http://http.us.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64)
+
+## Preparations
+
+```console
+cd ./boot/debian_12
+
+# Kernel
+wget -O vmlinuz http://http.us.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64/linux
+
+# InitRAM disk
+wget http://http.us.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64/initrd.gz
+
+# Grub loader
+wget http://http.us.debian.org/debian/dists/bookworm/main/installer-amd64/current/images/netboot/debian-installer/amd64/bootnetx64.efi -O grubx64.efi
```

Comment:
It's failing on:
```
python3 -m http.server --protocol HTTP/1.1 >> kokos.txt
192.168.122.16 - - [26/Feb/2024 13:59:49] "HEAD /grubx64.efi HTTP/1.1" 200 -
192.168.122.16 - - [26/Feb/2024 13:59:49] "GET /grubx64.efi HTTP/1.1" 200 -
----------------------------------------
Exception occurred during processing of request from ('192.168.122.16', 1863)
Traceback (most recent call last):
  File "/usr/lib64/python3.12/socketserver.py", line 692, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib64/python3.12/http/server.py", line 1311, in finish_request
    self.RequestHandlerClass(request, client_address, self,
  File "/usr/lib64/python3.12/http/server.py", line 672, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib64/python3.12/socketserver.py", line 761, in __init__
    self.handle()
  File "/usr/lib64/python3.12/http/server.py", line 438, in handle
    self.handle_one_request()
  File "/usr/lib64/python3.12/http/server.py", line 404, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/socket.py", line 707, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------

```

Doesn't matter if I run it under apache or just simple python server

Reply:
Not sure why it's calling `GET //grubx64.efi`

Reply:
In my testing with starting `bootnetx64.efi` (which chainloads `grubx64.efi`) I came to the conclusion that the grub image is built without the HTTP module, so it doesn't request the `grub.cfg` file. Though I haven't 100% verified it, I didn't see any request to my server. When I tried `insmod http` I did see `No DNS server configured` which leads me to https://bugzilla.redhat.com/show_bug.cgi?id=1842509.

I couldn't get my TFTP environment working for some reason and ran out of time experimenting.